Makefile

```diff
@@ -330,15 +349,6 @@ install-chart: build/toolchain/bin/helm$(EXE_EXTENSION)
 		--set openmatch.monitoring.stackdriver.enabled=true \
 		--set openmatch.monitoring.stackdriver.gcpProjectId=$(GCP_PROJECT_ID)
 
-install-demo-chart: build/toolchain/bin/helm$(EXE_EXTENSION)
```

Comment:
Should we provide a Makefile commands for open-match install along with the demo for people who wanna get started with using open-match?

Reply:
`make install-chart` will now install the demo by default.